data.json

```diff
@@ -593,6 +593,15 @@
     "dateAdded": "2020-09-03T15:43:00.000Z",
     "tags": ["stickers"]
   },
+  {
+    "name": "RoboPlay",
+    "difficulty": "medium",
+    "description": "Create a Discord bot or plugin with Robo.js - score 3 months of hosting, a month of Nitro, and a special edition T-Shirt!",
```

Comment:
Please adapt the wording to emphasize that swag will not be given to everyone but to the selected contributions only.

Reply:
@aslafy-z Just updated it. Does this new wording look better? :)